community-content/prediction_featurestore_integration/prediction_featurestore_integration.ipynb

```diff
@@ -0,0 +1,1470 @@
+{
```

Comment:
<p>May want to rethink how to populate FS. </p><p> </p><p>This is a lot of set up code that's distracting to the actual feature.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1371/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
Ingest features from an exported CSV instead.